test/MultiSig/transferablemultisig.test.ts

```diff
@@ -0,0 +1,175 @@
+import { assert } from 'chai';
+
+import * as Web3 from 'web3';
+
+import { SignHashArtifacts, TransferableMultiSig } from 'signhash';
+import { ContractContextDefinition } from 'truffle';
+
+import { TransferOwnershipCommand } from '../../multisig';
+
+import { assertThrowsInvalidOpcode } from '../helpers';
+import { MultiSigTestContext } from './context';
+
+declare const web3: Web3;
+declare const artifacts: SignHashArtifacts;
+declare const contract: ContractContextDefinition;
+
+const TransferableMultiSigContract = artifacts.require(
+  './MultiSig/TransferableMultiSig.sol'
+);
+
+contract('TransferableMultiSigContract', accounts => {
+  describe.skip('#ctor', () => {
```

Comment:
`skip`

Reply:
Skip usage already commented in other place